static/main.css

```
@@ -948,3 +948,44 @@ pre.arguments > div {
   text-decoration: none;
   color: #000;
 }
+
+.livelog {
+  max-height: 85vh;
+  overflow-y: scroll;
+}
+
+.livelog table tbody tr td {
+  white-space: normal;
+}
+
+.btn-primary a, .btn-primary a:visited {
+  color: #fff;
+}
+
+#download-logs {
+  margin: 5px 0 5px 0;
+}
+
+.livelog table td, .livelog table th {
+  padding:0;
+}
+
+.livelog table thead .livelog-timestamp {
```

you can use the nth-child psudo selector instead of naming them. 

```suggestion
#livelog th:nth-child(1) {
```

---

Yes - but I think the css is slightly easier to read/maintain with named elements rather than using nth-child in cases like this. I don't really mind either way though.

---

Absolut, judgement call. Just an idea.